src/projection-grid-core.js

```diff
@@ -30,14 +30,15 @@ export class ProjectionGridCore {
     };
   }
 
-  static createDefault() {
+  static createDefault(customDefaultContent = () => ({})) {
```

Comment:
We shouldn't add this parameter. Adding this particular projection after default content looks magic.
I suggest letting the projection-grid-react to insert the projection by itself. You can refer to the Vue version solution.
https://github.com/wewei/projection-grid-vue/blob/refactoring/src/projection-grid.vue#L12

Reply:
Isn't a more mysterious magic to play with the sequence of the builtin projections in react/vue config normalizer??

Reply:
We don't have the concept of normalizer. The defaultContent in React/Vue implementations is just plain projection. It shouldn't be treated special in core.

On the other hand, the semantic of the vue implementation is not to play with the projection sequence, but to append 1 projection after the defaultContent in core. No matter where it is. I feel it's OK for the React/Vue implementation to know there's a defaultContent in core, but the core shouldn't prepare a special slot for a normal projection. All passed in projections should be treated equal